CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 option(WITH_PTHREADS "Enable threading support through pthread. Highly recommended" ON)
 option(USE_XMLLINT "Run xmllint to test if data/cameras.xml is valid" ON)
 option(BUILD_TESTING "Build the testing tree." ON)
+option(WITH_ZLIB "Enable ZLIB support for DNG deflate support" ON)
```

Comment:
While there, please move this new line to after first two `option(WITH_`
Just OCD :)